src/main/kotlin/uk/gov/dluhc/printapi/database/mapper/CertificateSummaryDtoMapper.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.dluhc.printapi.database.mapper
+
+import org.springframework.stereotype.Component
+import uk.gov.dluhc.printapi.database.entity.Certificate
+import uk.gov.dluhc.printapi.database.entity.PrintRequest
+import uk.gov.dluhc.printapi.dto.CertificateSummaryDto
+import uk.gov.dluhc.printapi.dto.PrintRequestSummaryDto
+
+@Component
+class CertificateSummaryDtoMapper {
+
+    fun certificateToCertificatePrintRequestSummaryDto(certificate: Certificate): CertificateSummaryDto {
+        return CertificateSummaryDto(
+            vacNumber = certificate.vacNumber!!,
+            printRequests = toPrintRequests(certificate.printRequests)
+        )
+    }
+
+    private fun toPrintRequests(printRequests: MutableList<PrintRequest>): List<PrintRequestSummaryDto> =
+        printRequests.map { printRequest -> toPrintRequestSummaryDto(printRequest) }.toList()
+
```

Comment:
@Neil-Massey , I think we can safely omit .toList() at the end.

<pre>
printRequests.map { printRequest -> toPrintRequestSummaryDto(printRequest) } </pre>